reana-snakemake.yaml

```diff
@@ -0,0 +1,28 @@
+version: 0.8.0
+inputs:
+  files:
+    - ttbar_analysis_reana.ipynb 
+    - nanoaod_inputs.json
+    - fix-env.sh
+    - corrections.json
+    - Snakefile(EOS)
+    - file_merging.ipynb
+    - final_merging.ipynb
+    - prepare_workspace.py
+    - entrypoint.sh
+
+  directories:
```

Comment:
There are no directories "histograms" etc that would be inputs to the workflow, so I think we should delete them.
Only the workflow inputs should be presented in `inputs.directories` clause in `reana.yaml`.